storagenode/storagenodedb/database.go

```diff
@@ -117,16 +224,329 @@ func (db *DB) Close() error {
 		db.adb.Close(),
 
 		db.pieces.Close(),
-		db.info.Close(),
+
+		//db.versions.Close(),
```

Comment:
just curious why this is commented/why versions is handled differently?

Reply:
Oops I didn't mean to leave that commented out. Fixed!